extensions/notebook/src/extension.ts

```diff
@@ -129,6 +129,14 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 	const providedBookTreeViewProvider = new BookTreeViewProvider(appContext.apiWrapper, [], extensionContext, true, PROVIDED_BOOKS_VIEWID);
 	await providedBookTreeViewProvider.initialized;
 
+	azdata.nb.onDidChangeActiveNotebookEditor(e => {
```

Comment:
This will steal focus whenever any other viewlet is already open, right?

(i.e. say the connections viewlet is open and a user uses the previous/next buttons to navigate between notebooks. Wouldn't this always focus on the notebooks viewlet again?)

Reply:
Another scenario to test is the following:
- If a user navigates from one viewlet to the notebooks viewlet and a notebook that exists in the viewlet is opened, that notebook should be highlighted.

Reply:
yes, reveal will open the viewlet irrespective if the viewlet is not currently active.

Reply:
ok. how do we prevent that behavior?

Reply:
I believe passing in 'false' as the second parameter would stop this. Please correct me if I'm wrong.

Reply:
We don't need to focus the viewlet each time a notebook in the viewlet is active on the editor?